src/MacroTools/Commands/Share.cs

```diff
@@ -23,20 +24,43 @@ public sealed class Share : Command
     /// <inheritdoc />
     public override string Execute(player cheater, params string[] parameters)
     {
+      var cheaterTeam = cheater.GetTeam(); // Declare cheaterTeam once at the beginning
```

Comment:
```suggestion
      var cheaterTeam = cheater.GetTeam();
```
You can get rid of all these comments I reckon, it's clear what your code does just from reading it.